src/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,41 @@
+import './Categories.scss';
+import { Category } from '../Category/Category';
+
+import phones from '../../assets/categories/phones.png';
+import tablets from '../../assets/categories/tablets.png';
+import accessories from '../../assets/categories/accessories.png';
+
+export const Categories = () => {
+  return (
+    <section className="categories__section">
+      <div className="categories__container">
+        <h1 className="categories__title">
+          Shop by category
+        </h1>
+
+        <div className="categories">
```

Comment:
you use bem block inside block children,
```suggestion
    <section className="categories">
      <div className="categories__container">
        <h1 className="categories__title">
          Shop by category
        </h1>

        <div className="categories__section">
```